docs/source/usage.rst

```diff
@@ -0,0 +1,274 @@
+=====
+Usage
+=====
+
+This toolkit contains the tools to interface with HOLE_
+:cite:p:`Smart1993,Smart1996` to analyse an ion channel pore or transporter
+pathway :cite:p:`Stelzl2014`.
+
+Finding your hole executable
+----------------------------
+
+The first step is to find your HOLE executable. If you have installed HOLE
+through ``conda-forge``, then you can find the executable path by searching
+``which hole``. First ensure that you have activated the correct conda environment:
+
+.. code-block:: bash
+
+    $ conda activate mdahole2
+    $ which hole
+    /home/username/miniforge3/envs/mdahole2/bin/hole
+
+If you have installed HOLE from the original source, the executable
+may be in your home directory instead, e.g. a path such as ``~/hole2/exe/hole``.
+
+Throughout the rest of this documentation, we will assume that your executable
```

Comment:
Sorry I think I failed to communicate part of my initial question properly.

Our tests, and I would guess also the expected usage from our users, is that `hole` is found within $PATH. That is to say, it seems like our intended use case is that `executable` isn't explicitly passed by default? (please do correct me if that is an incorrect view of how we expect folks to do things).

In that case, I wonder if the wording here might either: a) tell folks to pass `executable` if they have hole defined in a non-standard way and then not show examples using `executable` or b) tell folks that you can just not pass `executable` in most cases and it'll work properly?

Reply:
Right, I see what you mean, thanks for the clarification. I've updated the docs as such.